.stylelintrc.js

```diff
@@ -14,5 +15,5 @@ module.exports = {
     ],
     'value-list-comma-newline-after': null, // let Prettier decide (when to put comma separated values on new lines)
   },
-  syntax: 'postcss',
+  syntax: 'scss',
```

Comment:
check out my config in the subscription component. I think it makes more sense for scss

Reply:
I can do that.

Also side-note: I’ve been rethinking the ordering rule, ever since this tweet: https://twitter.com/zachleat/status/1234583190679031809?s=20

Don’t get me wrong—I personally love it—but should the linter be that overbearing? Are our styling setups friendly enough for everyone? The only thing they help, to me, is a) easily-finding a CSS property, and b) preventing duplicate properties. But since we also have VRT it may just be a speedbump that doesn’t improve quality in a tangible way.

I guess it’s worth asking: does this slow people down (I’ve observed a lot of people push code, and not notice the lint error till much later), or speed people up (easier to find properties)? If the former, maybe we should remove just the ordering but keep everything else.

Reply:
personally I like the rational ordering. I often pause to think about what order properties should be in, so this makes the decision for me. Should it fail? probably not. Maybe just warn?

Also, I'm sure that I don't like alphabetical. Yes, a single property is easy to find but often if i'm changing one property, I'm also changing some related property.

Having all the grid/flex properties in one place means that I know all the grid properties are "here" and not scattered throughout the other properties. So if I switch from grid to block, I'm not scanning the rules for stray properties like `align-content` or `gap`, thery're all right there so I can select all the grid properties and delete.

Reply:
Yeah I can get down with that. Personally I still haven’t memorized the order, but you’re right I do compose in that order.

Also minor point: I see you’re using stylelint-config-standard-sass, but that seems to not be associated with stylelint-config-standard? It seems to be an old fork of [stylelint-config-recommended-scss](https://github.com/kristerkari/stylelint-config-recommended-scss), which seems more actively-maintained. Does that plugin enable any extra rules? In mine I didn’t seem to notice any difference (and stylelint as a project can be finicky so I try and install fewer plugins unless they add missing features).

Reply:
I'm not really sure tbh. I don't make mistakes so can't tell 🙃

Actually though, I'm sure what it's checking and what it isn't. So you're probably right